plugins/webhooks/index.js

```diff
@@ -77,7 +78,7 @@ const webhooksPlugin = {
                         for await (const chunk of request.payload) {
```

Comment:
probably it makes sense to get rid of custom logic for payload size and let hapi js do it in the payload configuration.
so I propose removing this and set
 payload: {
                    maxBytes: parseInt(pluginOptions.maxBytes, 10)